.eslintrc.js

```diff
@@ -1,9 +1,11 @@
 module.exports = {
   extends: ['@mate-academy/eslint-config-react-typescript', 'plugin:cypress/recommended'],
   rules: {
+    "react/require-default-props": "off",
```

Comment:
Consider adding a comment explaining why `react/require-default-props` rule is turned off. 🤔